### DISCUSSION TITLE
Header example with logged-in user details. notification and search bar

### DISCUSSION THREAD

**Discussion by satyajit-behera** [microsoft/fluentui-blazor#1000] [Q&A] [answered]
Do we have an example of Header with complete sample for user icon, details, notification and search bar? How can i create one such header using existing component?
Thanks in advance.

**Comment by dvoituron** [comment, marked as answer]
Which "Header" are you talking about? 
If it's FluentLayout / FluentHeader, you can find a complete example of its use in the demo site's source code.

https://github.com/microsoft/fluentui-blazor/blob/dev/examples/Demo/Shared/Shared/DemoMainLayout.razor
